### DISCUSSION TITLE
Releases interface would like to be able to quickly filter version types.

### DISCUSSION THREAD

**Discussion by SpiritLing** [community/community#63208] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/26963582/48811b61-c2c6-4529-9d30-07264fda84fb)

As you can see in the above figure, you want to quickly select the version type, otherwise you may have two dozen Pre-release versions and it will be difficult to find the last standard version.

**Comment by dmt0** [comment, marked as answer]
Duplicate of https://github.com/orgs/community/discussions/6108 ?
